frontend/scss/components/organisms/header.scss

```diff
@@ -81,53 +92,256 @@
           @include txt-lrg;
         }
       }
-
-      &-sub-title {
```

Comment:
header.j2 is not the only file using this CSS. This selector is needed inside frontend/hbs/preview-email.hbs